Makefile

```diff
@@ -64,8 +64,8 @@ deploy.booking: ##=> Deploy booking service using SAM
 			--parameter-overrides \
 				BookingTable=$${BOOKING_TABLE_NAME} \
 				FlightTable=$${FLIGHT_TABLE_NAME} \
-				CollectPaymentFunction=/service/payment/collect-function/$${AWS_BRANCH} \
-				RefundPaymentFunction=/service/payment/refund-function/$${AWS_BRANCH} \
+				CollectPaymentFunction=/$${AWS_BRANCH}/service/payment/collectFunction \
```

Comment:
I’d probably go for a structure of /service/payment/function/collect since it’s narrowing the scope at each /